js/views/dataset.vue

```diff
@@ -168,17 +173,17 @@ export default {
         }
     },
     components: {
-        sbox: require('components/containers/small-box.vue'),
         dataset: require('components/dataset/details.vue'),
         quality: require('components/dataset/quality.vue'),
         chart: require('components/charts/widget.vue'),
         resources: require('components/dataset/resource/list.vue'),
         followers: require('components/follow/list.vue'),
         wmap: require('components/widgets/map.vue'),
-        issues: require('components/issues/list.vue'),
-        discussions: require('components/discussions/list.vue'),
-        community: require('components/dataset/communityresource/list.vue'),
+        CommunityList,
```

Comment:
yes, it is :) But it's very well documented:
- https://vuejs.org/guide/components.html#Assets-Naming-Convention
- https://vuejs.org/guide/components.html#camelCase-vs-kebab-case

This is mainly du to the fact that on one side, html is case insensitive and recommend using dash into WebComponents name and properties and on the other side, dash is not legit in JavaScript naming and CamelCase is the convention.